packages/director/src/screenshots/s3/s3.ts

```diff
@@ -74,5 +82,47 @@ export const getVideoUploadUrl = async (
   getUploadUrl({
     key: `${sanitizeS3KeyPrefix(S3_VIDEO_KEY_PREFIX)}${key}.mp4`,
     ContentType: VideoContentType,
-    Expires: UPLOAD_EXPIRY_SECONDS,
+    Expires: S3_UPLOAD_EXPIRY_SECONDS,
+  });
+
+// Signed downloads
+export const getDownloadUrl = async ({
+  key,
+  Expires = S3_DOWNLOAD_EXPIRY_SECONDS,
+}: GetDownloadURLParams): Promise<S3SignedDownloadResult> => {
+  const s3Params = {
+    Bucket: S3_BUCKET,
+    Key: key,
+    Expires
+  };
+
+  return new Promise((resolve, reject) => {
```

Comment:
The signed download URL will be persisted to the DB with the expired signature. I think the signing should be done on the API each time we retrieve the url